api/v1alpha1/customdomain_types.go

```diff
@@ -43,6 +43,70 @@ type CustomDomainSpec struct {
 	RouteSelector *metav1.LabelSelector `json:"routeSelector,omitempty"`
 }
 
+// AWSLoadBalancerType is the type of AWS load balancer to instantiate.
```

Comment:
It looks like this code was taken from [operator/v1/types_ingress](https://github.com/openshift/api/blob/master/operator/v1/types_ingress.go#L576-L595). Can we just import this to avoid duplication?

See how this is done here: https://github.com/openshift/custom-domains-operator/blob/89f56553c1f386e96257071d4fcd744547aec220/main.go#L35

Reply:
Thanks for checking! I updated the code.